### ISSUE TITLE
Polygon Default: Style by Value > Category // Quantity

### ISSUE THREAD

**Issue by makella** [CartoDB/cartodb#8820]
@javisantana @saleiva 
#### Defaults
- Classification Method: Quantiles
- Number of Classes: 5
- Default Ramp: `BluGrn`
- Default Styling:  
  **note there are some cartography specific cartocss properties please keep them**

```
#layer {
  polygon-fill: ramp([percent_poverty], (#1d4f60, #2d7974, #4da284, #80c799, #c4e6c3), quantiles);
  polygon-opacity: 0.9;
  polygon-gamma: 0.5;
  line-width: 0.5;
  line-color: #fff;
  line-opacity: 0.5;
  line-comp-op: soft-light;
}
```
- Example:
  ![screen shot 2016-07-12 at 2 51 10 pm](https://cloud.githubusercontent.com/assets/1566273/16783167/3c866cba-4840-11e6-9bfc-6492f90baf4a.png)

![screen shot 2016-07-12 at 2 51 21 pm](https://cloud.githubusercontent.com/assets/1566273/16783173/428a6fee-4840-11e6-851c-2d84ae4d0531.png)
#### Defaults
- Default Ramp: `Vivid`
  **note: i would like to switch this to `Bold` if auto styling is not going to be available**
- Default Styling:  
  **note there are some cartography specific cartocss properties please keep them**

```
#layer {
  polygon-fill: ramp([geoid],(#E58606,#5D69B1,#52BCA3,#99C945, #2F8AC4,#24796C,#cc2d7f,#764e9f,#ed645a,#CC61B0));
  polygon-opacity: 0.9;
  polygon-gamma: 0.5;
  line-width: 0.5;
  line-color: #fff;
  line-opacity: 0.5;
  line-comp-op: soft-light;
}
```
- Examples:  
  ![screen shot 2016-07-12 at 3 08 38 pm](https://cloud.githubusercontent.com/assets/1566273/16783813/f86ebcaa-4842-11e6-8d0b-72b431a8e7e8.png)

![screen shot 2016-07-12 at 3 08 57 pm](https://cloud.githubusercontent.com/assets/1566273/16783825/0ae20b4e-4843-11e6-9b98-1b5e0a510d24.png)

**Comment by makella** [comment]
this will change per https://github.com/CartoDB/cartodb/issues/8819#issuecomment-232369412

since we can't use anything other than very basic defaults.

**Comment by makella** [comment]
@javisantana updates here based on the simple styling conversation from this morning.

**Just so you know**
- adding the `::outline` attachment makes the lines so much more prominent on the map even with such a small line width and opacity. But if we must, ok. If we don't have to, please don't
- I added a comment with an example here: https://github.com/CartoDB/cartodb/issues/7723

### Numeric
#### Defaults 
- Classification Method: Quantiles
- Number of Classes: 5
- Default Ramp: `BluGrn`
- Default Styling:   

```
#layer { 
  polygon-fill: ramp([percent_poverty], (#1d4f60, #2d7974, #4da284, #80c799, #c4e6c3), quantiles);
  polygon-opacity: 0.9;
  line-color: #fff;
  line-width: 0.5;
  line-opacity: 0.5;
}
```

### Categories
#### Defaults
- Default Ramp: `Vivid`  
- Default Styling:  

```
#layer {
  polygon-fill: ramp([geoid],(#E58606,#5D69B1,#52BCA3,#99C945, #2F8AC4,#24796C,#cc2d7f,#764e9f,#ed645a,#CC61B0));
  polygon-opacity: 0.9;
  line-color: #fff;
  line-width: 0.5;
  line-opacity: 0.5;
}
```

**Comment by makella** [comment]
hello,

I understand that getting defaults set up takes more time than is available right now. 

I am worried that every map will have the `Burg` ramp by default made by our users. There is nothing wrong with it, but the `BluGrn` ramp is more subtle and as a default (in my opinion) is better suited.

To get around the not enough time right now hurdle, I figured that I could reorder the ramps in CartoColors so the `BluGrn` ramp would be at the top and therefore draw as the default.

I arranged all the other ramps according to that so they follow some order within their color families.

Can my CartoColor PR be merged as a secondary way to get in this default?
https://github.com/CartoDB/CartoColor/pull/15
